spec/queries/my_facilities_query_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe MyFacilitiesQuery do
+  context 'utility methods' do
+    let!(:active_facility) { create(:facility) }
+    let!(:inactive_facility) { create (:facility) }
+    let!(:inactive_facility_with_zero_bps) { create (:facility) }
+    let!(:blood_pressures_for_active_facility) { create_list(:blood_pressure, 10, facility: active_facility, recorded_at: Time.now) }
+    let!(:blood_pressures_for_inactive_facility) { create_list(:blood_pressure, 9, facility: inactive_facility, recorded_at: Time.now) }
+    let!(:encounters) do
+      (blood_pressures_for_active_facility + blood_pressures_for_inactive_facility).each { |record|
+        create(:encounter, :with_observables, patient: record.patient, observable: record, facility: record.facility) }
+    end
+
+    describe ".inactive_facilities" do
+      it 'should return only inactive facilities' do
+        facility_ids = [active_facility.id, inactive_facility.id, inactive_facility_with_zero_bps.id]
+        expect(MyFacilitiesQuery.inactive_facilities(Facility.where(id: facility_ids))).to match_array([inactive_facility, inactive_facility_with_zero_bps])
```

Comment:
Since this test case is covering all the facilities created in this context, can we just `expect(MyFacilitiesQuery.inactive_facilites).to match_array...`?

Reply:
Depending on our decision here, could we add another test for the other scenario? Effectively, I'd love to have one test for what happens when we pass in a scope, and one test for what happens when we don't (defaulting to `.all`)

Reply:
This isn't covering all the facilities, the factories create some facilities implicitly.